tests/Output/ColorTest.php

```diff
@@ -49,36 +49,39 @@ public function test_invalid_custom_style()
 
     public function test_colors()
     {
-        $c = new Color;
+        $c = new Color();
 
         $this->assertSame("\nabc\n", $c->colors('<eol>abc</eol>'));
         $this->assertSame("\033[0;31mRed\033[0m", $c->colors('<red>Red</end>'));
         $this->assertSame("\033[1;31mBoldRed\n\033[0m", $c->colors('<boldRed>BoldRed<eol/></end>'));
         $this->assertSame("\033[0;36;42mBgGreenCyan\033[0m\n", $c->colors('<bgGreenCyan>BgGreenCyan</end><eol>'));
-        $this->assertSame("\033[0;31mRed\033[0m\nNormal\n\033[1;37mBOLD\033[0m", $c->colors("<red>Red</end>\r\nNormal\n<bold>BOLD</end>"));
+        $this->assertSame(
+            "\033[0;31mRed\033[0m\nNormal\n\033[1;37mBOLD\033[0m",
+            $c->colors("<red>Red</end>\r\nNormal\n<bold>BOLD</end>")
+        );
     }
 
     public function test_magic_call()
     {
-        $this->assertSame("\033[1;37mline\033[0m", (new Color)->bold('line'));
+        $this->assertSame("\033[1;37mline\033[0m", (new Color())->bold('line'));
 
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage('Text required');
 
-        (new Color)->bgRed();
+        (new Color())->bgRed();
     }
 
     public function test_magic_call_color()
     {
-        $this->assertSame("\033[0;35mpurple\033[0m", (new Color)->purple('purple'));
+        $this->assertSame("\033[0;35mpurple\033[0m", (new Color())->purple('purple'));
```

Comment:
is it possible to exclude this form?

(new Klass)->method() looks neater

Reply:
Sure